docs/versioned/admin/admin-overview.md

```diff
@@ -50,59 +112,40 @@ Knative configurations are performed by the following methods:
-
-Configurations for security encryptions:
-
-- [cert-manager](../serving/encryption/configure-certmanager-integration.md)
-- [External domains](../serving/encryption/external-domain-tls.md)
-- [Local domains](../serving/encryption/cluster-local-domain-tls.md)
-- [system-internal](../serving/encryption/system-internal-tls.md)
-
-Configurations for extensions:
-
-- [Kafka Broker features](../serving/encryption/system-internal-tls.md)
-- [Sugar Controller](../eventing/configuration/sugar-configuration.md)
-
-Configurations for flagging features:
-
-- [Serving features](../serving/configuration/feature-flags.md)
-- [Eventing features](../eventing/features/README.md)
-
-## Authorizations
-
-You can grant developers access to additional resources related to their namespace in other services, such as observability, logs, metrics, tracing, and dashboards.
+Knative uses Kubernetes YAML manifests to define and configure system components. These manifests include core resources, custom resource definitions (CRDs), and extensibility features. As with Kubernetes, these configuration resources are declarative and can be managed using the `kubectl` CLI tool or with continuous delivery tools.
 
-## Upgrades
+The following sections provide an overview of the current configurations procedures of interest to Administrators. You make these configurations using `kubectl`, with some procedures applying YAML files.
+
+### Serving configurations
+
+| Configuration | ConfigMap | Description |
+| -- | --- | --- |
+| [Default configurations](../serving/configuration/config-defaults.md) | `config-defaults` | Default resource values such as performance, hardware, and storage settings. |
+| [Deployment resources](../serving/configuration/deployment.md) | `config-deployment` | Kubernetes deployment resources that back Knative services. |
+| [Domain names](../serving/using-a-custom-domain.md) | `config-domain` | Configure and publish domains. |
+| [High-availability](../serving/config-ha.md) | NA | Configure ensure that APIs stay operational if a disruption occurs. |
+| [Garbage collection](../serving/revisions/revision-admin-config-options.md) | `config-gc` | Disable and enable collection and set retention time values. |
+| [Ingress gateway](../serving/setting-up-custom-ingress-gateway.md)| `config-istio` | For new clusters, you can configure your own gateway and underlying service. |
+| [Istio authorization](../serving/istio-authorization.md) | NA | Grant authorization to your deployed Knative services. |
+| [Namespace exclusion from webhook](../serving/webhook-customizations.md) | NA | For performance concerns during an upgrade. |
+| [Rollout duration for revisions](../serving/configuration/rolling-out-latest-revision-configmap.md) | `config-network` | Adjust rollout durations to accommodate longer request queues. |
+| [Security - Certificates](../serving/encryption/configure-certmanager-integration.md) | NA | Describes how to manage automatic certificate provisioning. |
```

Comment:
```suggestion
| [Security - Certificates](../serving/encryption/configure-certmanager-integration.md) | `config-certmanager` | Describes how to manage automatic certificate provisioning. |
```

The CertManager integration has extra requirements, but it _does_ involve editing `config-certmanager`, so it's worth linking that here.